.pre-commit-config.yaml

```diff
@@ -0,0 +1,9 @@
+repos:
+    - repo: https://github.com/pre-commit/mirrors-prettier
+      rev: "v2.2.1" # Use the sha or tag you want to point at
+      hooks:
+          - id: prettier
+    - repo: https://github.com/pre-commit/mirrors-eslint
+      rev: "v7.23.0" # Use the sha / tag you want to point at
+      hooks:
+          - id: eslint
```

Comment:
Specify the types (file extensions) we need to lint.
Reference: [eslint](https://github.com/pre-commit/mirrors-eslint), [prettier](https://github.com/pre-commit/mirrors-prettier)

Reply:
I have specified file extension for eslint, but I think prettier is supposed to run on all files.

Reply:
You can specify types in prettier as

```yaml
...
  -   id: prettier
      types: [javascript]
...
```

Reply:
Please also specify `--fix` as `args` in eslint hook

Reply:
By default, it would run on all files. If you still need me to specify just for js and ts, I can do that.

Reply:
Please specify it run on only `js` and `ts` files.

Reply:
Please review.